sdk/nodejs/src/client.ts

```diff
@@ -455,4 +456,16 @@ export class DevCycleClient<
         this.eventQueue.cleanup()
         clearInterval(this.bucketingTracker)
     }
+
+    setClientCustomData(clientCustomData: DVCCustomDataJSON): void {
```

Comment:
@JamieSinn changing this to be sync like all the other SDKs to work with the test harness

Reply:
kk